Shopping COVID/Assets/Scripts/GameSceneController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameSceneController : MonoBehaviour
```

Comment:
Drop candidate, will look into it on the level transitions ($11s Add transitions between levels) branch